openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx

```diff
@@ -358,6 +359,31 @@ const ContainerPage = () => {
       showErrorToast(error as AxiosError);
     }
   };
+  const handleUpdateDisplayName = async (data: EntityName) => {
+    if (isUndefined(containerData)) {
+      return;
+    }
+    try {
+      const { displayName, version } = await handleUpdateContainerData({
+        ...containerData,
+        displayName: data.displayName,
+      });
+
+      setContainerData((prev) => {
+        if (isUndefined(prev)) {
+          return;
+        }
+
+        return {
+          ...prev,
+          displayName,
+          version,
```

Comment:
Any reason to update version too?

Reply:
Yes we show updated version number on version button